addons/web/static/src/js/views/control_panel/control_panel_model.js

```diff
@@ -1114,7 +1116,7 @@ var ControlPanelModel = mvc.Model.extend({
                         domain: favorite.domain,
                         groupBys: groupBys,
                         // we want to keep strings as long as possible
-                        context: JSON.stringify(context),
+                        context: favorite.context,
```

Comment:
I think we can simply do this (and remove the comment):
```suggestion
                        context: context,
```
Keeping the context as a string is useful when we can't evaluate it directly (because there is no evaluation context yet). Here, we can evaluate it (and we do). I ran the 'views' part of the QUnit test and all tests pass.

Reply:
Indeed. It looks like it's the best solution as `pyUtils.eval()` doesn't try to evaluate a context given as a JS object and uses it as is.